client/src/java/com/zimbra/client/ZContact.java

```diff
@@ -96,10 +97,12 @@ public long getLength() {
 
     /* Note for contacts, use a more limited set of flags than in ZItem */
     public enum Flag {
-        flagged('f'),
-        attachment('a');
+        flagged(-6, 'f'),
+        attachment(-2, 'a');
 
         private char mFlagChar;
+        final int id;
+        final int bitmask;
```

Comment:
Ah, looks like it is. I think when I looked for the existing code initially, I accidentally looked at at the ZItem class in the old zm-store repo, which doesn't have the toBitmask method.